SPECS/libmicrohttpd/libmicrohttpd.spec

```diff
@@ -1,21 +1,29 @@
-Summary:        Lightweight library for embedding a webserver in applications
 Name:           libmicrohttpd
-Version:        0.9.76
-Release:        1%{?dist}
-License:        LGPLv2+
 Vendor:         Microsoft Corporation
-Distribution:   Mariner
-URL:            https://www.gnu.org/software/libmicrohttpd/
+Distribution:   Microsoft Azure Linux
+Version:        0.9.77
+Release:        3%{?dist}
+Summary:        Lightweight library for embedding a webserver in applications
+
+# * COPYING says that some main sources are only under LGPL-2.1-or-later
+#   and the rest is dual licensed under LGPL-2.1-or-later OR GPL-2.0-or-later WITH eCos-exception-2.0.
+# * Some docs are under GFDL-1.3-no-invariants-or-later.
+# * Tests and some parts of the build system are under other licenses but they are NOT shipped.
+#License:        LGPL-2.1-or-later AND (LGPL-2.1-or-later OR GPL-2.0-or-later WITH eCos-exception-2.0) AND GFDL-1.3-no-invariants-or-later
```

Comment:
Can you say what was the issue with the tooling? We should log a bug for this.

Reply:
i dont actually remember exactly (there have been quite a lot of toolkit/prcheck failures i had to work around); i updated the code to go back to the complex license and pushed it, so if it passes pr checks and test runs then i guess it's ok :-)